fe/fe-core/src/main/java/com/starrocks/sql/analyzer/InsertAnalyzer.java

```diff
@@ -110,11 +92,6 @@ public static void analyze(InsertStmt insertStmt, ConnectContext session) {
             throw unsupportedException("Only support insert into olap table or mysql table or iceberg table");
         }
 
-        if (table instanceof IcebergTable && CatalogMgr.isInternalCatalog(catalogName)) {
```

Comment:
why remove this validation？

Reply:
Is it possible to create an iceberg table in a non-iceberg catalog?

Will put it back if it's necessary.